docs/tutorials/tqdm_progress_bar.ipynb

```diff
@@ -0,0 +1,465 @@
+{
```

Comment:
<p>Mind removing these warnings from the notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/addons/pull/610/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
Will do, thanks!